src/task_4_2.py

```diff
@@ -0,0 +1,13 @@
+# Дан список целых чисел. Подсчитать сколько четных чисел в списке
+print("Используем цикл While: ")
+num = [2, -12, -7, 16, 8, 6]
+k = 0
+while num:
+    if num.pop() % 2 == 0:
+        k += 1
+print(f"Количество четных чисел: k = {k}")
+print("\nИспользуем цикл For: ")
+for i in range(len(num)):
```

Comment:
так как i здесь все равно не используется то можешь заменить его на _

```suggestion
for _ in range(len(num)):
```